apps/omg_watcher/lib/web/controllers/fallback.ex

```diff
@@ -18,28 +18,65 @@ defmodule OMG.Watcher.Web.Controller.Fallback do
   """
 
   use Phoenix.Controller
+  alias OMG.RPC.Web.Error
 
-  alias OMG.Watcher.Web.Serializers
-
-  def call(conn, :not_found) do
-    data = %{
-      object: :error,
-      code: :endpoint_not_found,
-      description: "Endpoint not found"
+  @errors %{
+    exit_not_found: %{
+      code: "challenge:exit_not_found",
+      description: "The challenge of particular exit is impossible because exit is inactive or missing"
+    },
+    utxo_not_spent: %{
+      code: "challenge:utxo_not_spent",
+      description: "The challenge of particular exit is impossible because provided utxo is not spent"
+    },
+    transaction_not_found: %{
+      code: "transaction:not_found",
+      description: "Transaction doesn't exist for provided search criteria"
+    },
+    utxo_not_found: %{
+      code: "exit:invalid",
+      description: "Utxo was spent or does not exist."
+    },
+    tx_for_input_not_found: %{
+      code: "in_flight_exit:tx_for_input_not_found",
+      description: "No transaction that created input."
+    },
+    econnrefused: %{
+      code: "get_status:econnrefused",
+      description: "Cannot connect to the Ethereum node."
     }
+  }
+
+  def call(conn, Route.NotFound), do: json(conn, Error.serialize(:endpoint_not_found, "Endpoint not found"))
```

Comment:
why Route.NotFound  and not :not_found ?

Reply:
Simple `:not_found` can be returned from API, e.g. when there is no tx of given hash... I want to avoid ambiguity